### ISSUE TITLE
Unsupported event type: issue_comment

### ISSUE THREAD

**Issue by kevinsmith** [daun/playwright-report-summary#203]
We're triggering playwright runs using slash commands in our PR comments, which means the action is configured like this:

```yaml
on:
  issue_comment:
    types: [created]

jobs:
  test:
    if: ${{ github.event.issue.pull_request && startsWith(github.event.comment.body, '/playwright') }}
```

Is it possible to get this report summary to support this event type?

**Comment by daun** [comment]
Hi. That's just a warning to say the action is meant to be run on pull requests. It should still go through and run as configured in the latest version. Is it not running for you or does the warning bother you?

**Comment by kevinsmith** [comment]
It doesn't run at all. The next line it prints out is...

> No PR associated with this action run. Not posting a check or comment.

**Comment by daun** [comment]
I see. In that case the action doesn‘t know how to get the contextual data required to display the latest commit message and hash. If you can share your whole setup so I can try and replicate it, I can look into how to get that data.

**Comment by daun** [comment]
I've released [v3.5.0](https://github.com/daun/playwright-report-summary/releases/tag/v3.5.0) of the action that should know how to identify the PR in question when triggered by a comment. Please try it out and let me know how it goes.

**Comment by kevinsmith** [comment]
It worked! One thing though, it looks like the commit SHA it references in the comment is actually the default branch's HEAD, not the pull request's HEAD.

**Comment by daun** [comment]
Right. Fixed in 3.5.1 to display the correct commit sha.

**Comment by kevinsmith** [comment]
That did it! Thanks so much.
